sw/device/lib/testing/rv_core_ibex_testutils.c

```diff
@@ -12,18 +12,18 @@
 bool rv_core_ibex_testutils_is_rnd_data_valid(
     const dif_rv_core_ibex_t *rv_core_ibex) {
   dif_rv_core_ibex_rnd_status_t rnd_status;
-  CHECK_DIF_OK(dif_rv_core_ibex_get_rnd_status(rv_core_ibex, &rnd_status));
-  return rnd_status & kDifRvCoreIbexRndStatusValid;
+  dif_result_t res = dif_rv_core_ibex_get_rnd_status(rv_core_ibex, &rnd_status);
+  return res == kDifOk && rnd_status & kDifRvCoreIbexRndStatusValid;
```

Comment:
I suggest parenthesis here and maybe can explicit test:
 `res == kDifOk && (rnd_status & kDifRvCoreIbexRndStatusValid) != 0`

Reply:
first comment is valid tho

Reply:
Done